app/services/save_auction.rb

```diff
@@ -4,10 +4,20 @@ def initialize(auction)
   end
 
   def perform
-    auction.save
+    saved = auction.save
+    schedule_auction_ended_job(saved)
+
+    saved
   end
 
   private
 
   attr_reader :auction
+
+  def schedule_auction_ended_job(saved)
+    if saved
+      job = AuctionEndedJob.new(auction.id)
+      Delayed::Job.enqueue(job, run_at: auction.ended_at)
```

Comment:
do we want to add logic for updating this job if `ended_at` changes or is that for a separate PR?